stylesheet/main.css

```diff
@@ -0,0 +1,84 @@
+:root {
+    --color-text:#ffffff;
+    --color-text-primary:var(--color-text);
+    --color-background:#ffffff;
+    --color-background-primary:var(var(--color-background));
+
+    --font-family-nav:'Montserrat', sans-serif;
+    --font-family-text:'Playfair Display', serif;
+}
+
+.header {
+    display: grid;
+
+    align-items: center;
+
+    grid-template-rows: 100px;
+
+    grid-template-areas: 'logo nav phone';
+    
+    z-index: 1;    
+}
+
+.nav {
+    display: inline-block;
+
+    color: var(--color-text);
+
+    margin-left:100px;
+
+    z-index: 1;
+
+    font-family: var(--font-family);
+}
+
+.main {
+    background-color: var(--color-background-primary);
+    
+    margin: 0;
+
+    font-family: var(--font-family-text);
+}
+
+.nav--ul {
+    display: flex;
+
+    flex-direction: row;
+}
+
+.nav--list {
+    margin-left:40px;
+
+    font-size: 13px;
+
+    list-style: none;
+
+    text-transform: capitalize;
+
+    justify-content: center;
+}
+
+.phone {
+    font-weight: bolder;
+
+    font-size: 17px;
+
+    grid-area: phone; 
+
+    color: var(--color-text);
+
+    font-family: var(--font-family-nav);
+
+    margin-left: 20px;
+    margin-right: 0px;
+}
+
+a {
```

Comment:
de regula cand scriem CSS incercam sa punem stilurile pentru elementele HTML "chioare" at the top .. pt ca stilu asta se aplica pe toate linkurile si sa fie usor de detectat